README.md

```diff
@@ -24,31 +24,33 @@ Prior to JupyterHub 2.0, the `jupyterhub-idle-culler` required full administrati
 in order to have sufficient permissions to stop servers on behalf of users.
 
 JupyterHub 2.0 introduces [scopes][] to allow for more fine-grained permission control.
-This means that the cull-idle service does not need full administrative privileges anymore.
+This means that the configured culler service does not need full administrative privileges anymore.
```

Comment:
I thought it was hard to keep track of a registered service name like idle-culler, a role name like cull-idle, and a cli name jupyterhub-idle-culler --- so I opted to name them all jupyterhub-idle-culler and mentions here "the configured culler service" to decouple from whatever name is used.

Reply:
This is always hard for me! I see both different names for each kind of thing, and using the same name for everything as sources of confusion. I like what you have here.

To me:
- Downside of different names: more names to keep track of
- Downside of the same name: it's harder to see which ones *matter* that they match (e.g. the role name and service name don't have to be related, but the service name *does* have to match the name in the services list in the role)

Reply:
I think my main objection isn't naming them different things, but that the name `cull-idle` as a service name and `idle-culler` as a role name felt too arbitrary.

I have seen two practices when naming k8s resources, where the dominant have become to use the same name without adding a suffix of `-<type>`, but adding a suffix have also been used for the purpose of clarifying the resource type.

Do you think it would be a better compromise to have a name for the role as `jupyterhub-idle-culler-role` and a name for the service as `jupyterhub-idle-culler-service` ? Then, it would be quite clear I think, and it would not make me confused about what `cull-idle` (service) vs `idle-culler` (role) is for example.

Reply:
I pushed such changes to this PR!

Reply:
I actually have a general preference to avoid redundant names (e.g. service hub-service, pod hub-pod, role my-role), but I think it's a fine compromise in this case, and there's no need to spend too much time picking a few strings in an example!